torch/onnx/symbolic_caffe2.py

```diff
@@ -187,3 +185,32 @@ def reshape(g, input, shape):
     output = g.op("_caffe2::Int8Reshape", input, shape, **kwargs)
     sym_help._quantized_ops.add(output)
     return output
+
+@parse_args('v', 'v', 'v', 'v', 'i')
+def slice(g, input, dim, start, end, step):
+    if input not in sym_help._quantized_ops:
+        from torch.onnx.symbolic_opset9 import slice
+        return slice(g, input, dim, start, end, step)
+
+    if step != 1:
+        raise RuntimeError("ONNX quantized slice export only works for step 1.")
+    start = sym_help._parse_arg(start, 'i')
+    end = sym_help._parse_arg(end, 'i')
+    dim = sym_help._parse_arg(dim, 'i')
+
+    # Dummy values as C2 op expects them to be passed.
+    start_list = [0]
```

Comment:
If we don't provide starts and ends, will that cause crash or not? If so, we probably want to relax the constrains a bit.

Reply:
Yes, it will crash. I didn't want to change existing behavior for existing ops so didn't modify that. @houseroad do you think I should?

Reply:
I think it's fine, it should backward compatible. Let's move the ENFORCE on starts/ends to the else branch of checking existence of `dim`.